### DISCUSSION TITLE
errors in a vpn app flutter

### DISCUSSION THREAD

**Discussion by NegarTavakol** [community/community#121031] [Programming Help]
### Select Topic Area

Question

### Body

hi there , I'm trying to write a Free Vpn App in flutter . I am Iranian as you know Iran has very Strong filtering so I can not use  applications like whats app for talking with my family so I decided to write a Vpn App for it ,cause every day new VpnApps suspended in Iran , I have some issue in my app like : 
The following MissingPluginException was thrown while activating platform stream on channel
vpnStage:
MissingPluginException(No implementation found for method listen on channel vpnStage)

When the exception was thrown, this was the stack:
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:332:7)
<asynchronous suspension>
#1      EventChannel.receiveBroadcastStream.<anonymous closure> (package:flutter/src/services/platform_channel.dart:676:9)
<asynchronous suspension>
════════════════════════════════════════════════════════════════════════════════════════════════════

Another exception was thrown: MissingPluginException(No implementation found for method listen on channel vpnStatus)
    I try to fix them but my knowledge is weak and need to getting help to fix them . Is there any person to solve or guide me to fix them?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @NegarTavakol ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
